deps/wazuh_testing/wazuh_testing/qa_docs/lib/config.py

```diff
@@ -32,153 +36,194 @@ def __init__(self, *args):
+
         if not 'Group files' in self._config_data:
-            logging.error("Config group files is empty")
-            raise Exception("Config group files is empty")
+            raise QAValueError('Config include paths are empty', Config.LOGGER.error)
+
         self.group_files = self._config_data['Group files']
 
-    def _read_function_regex(self):
+    def __read_function_regex(self):
         """
         brief: Reads from the config file the regexes used to identify a test method.
         """
+        Config.LOGGER.debug('Reading the regular expressions to include test methods from config file')
+
         if not 'Function regex' in self._config_data:
-            logging.error("Config function regex is empty")
-            raise Exception("Config function regex is empty")
+            raise QAValueError('Config function regex is empty', Config.LOGGER.error)
+
         self.function_regex = self._config_data['Function regex']
 
-    def _read_ignore_paths(self):
+    def __read_ignore_paths(self):
         """
         brief: Reads from the config file all the paths to be excluded from the parsing.
         """
+        Config.LOGGER.debug('Reading the paths to be ignored from config file')
+
         if 'Ignore paths' in self._config_data:
             ignore_paths = self._config_data['Ignore paths']
+
             for path in ignore_paths:
                 self.ignore_paths.append(os.path.join(self.project_path, path))
 
-    def _read_module_fields(self):
+    def __read_module_fields(self):
         """
         brief: Reads from the config file the optional and mandatory fields for the test module.
         """
+        Config.LOGGER.debug('Reading mandatory and optional module fields from config file')
+
         if not 'Module' in self._config_data['Output fields']:
-            logging.error("Config output module fields is missing")
-            raise Exception("Config output module fields is missing")
+            raise QAValueError('Config output module fields is missing', Config.LOGGER.error)
+
         module_fields = self._config_data['Output fields']['Module']
+
         if not 'Mandatory' in module_fields and not 'Optional' in module_fields:
-            logging.error("Config output module fields are empty")
-            raise Exception("Config output module fields are empty")
+            raise QAValueError('Config output module fields are empty', Config.LOGGER.error)
+
         if 'Mandatory' in module_fields:
             self.module_fields.mandatory = module_fields['Mandatory']
+
         if 'Optional' in module_fields:
             self.module_fields.optional = module_fields['Optional']
 
-    def _read_test_fields(self):
+    def __read_test_fields(self):
         """
         brief: Reads from the config file the optional and mandatory fields for the test functions.
         """
+        Config.LOGGER.debug('Reading mandatory and optional test fields from config file')
```

Comment:
```suggestion
        Config.LOGGER.debug('Reading mandatory and optional test fields from the config file')
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/commit/86e5045beb49f8e91dbb42b6e59f8d3164a57523